setup.cfg

```diff
@@ -25,37 +25,40 @@ keywords = pytorch, deep learning, machine learning, remote sensing, satellite i
 
 [options]
 install_requires =
-    einops<0.5
-    # fiona 1.5+ required for fiona.transform module
-    fiona~=1.5
+    # einops 0.3+ required for einops.repeat
+    einops>=0.3,<0.5
+    # fiona 1.8+ required for reading empty files
+    fiona>=1.8,<2
```

Comment:
Our fake data for SpaceNet 4 contains empty `labels.geojson` files. Older versions of fiona raise a different error message than the one we test for.

TODO: can we just add real data to these files to support older fiona?

Reply:
Apparently the actual dataset occasionally contains empty files so this is on purpose.